### DISCUSSION TITLE
multi_scale arg DOCS

### DISCUSSION THREAD

**Discussion by dimidagd** [ultralytics/ultralytics#8293] [Docs]
I have discovered that the train arg `multi_scale`  is not included in the DOCS [train args](https://docs.ultralytics.com/modes/train/#arguments). Is this left-out on purpose? Is the use of multi_scale flag not encouraged?

**Comment by dimidagd** [comment]
Closing as posted now in https://github.com/orgs/ultralytics/discussions/2354#discussioncomment-8515352

**Reply by glenn-jocher** [reply]
@dimidagd thank you for bringing this to our attention! The `multi_scale` argument is indeed part of our training capabilities, and it seems it was inadvertently omitted from the documentation. We appreciate your vigilance in spotting this oversight. The documentation will be updated to reflect this feature. In the meantime, if you wish to use multi-scale training, you can do so by adding `multi_scale=True` to your training command. If you have any further questions or need assistance, please don't hesitate to ask in the discussions or open a new issue. Happy training! 😊🚀
